### PR TITLE
Items

### PR DESCRIPTION
**Pull request by FidaaHersh**
Finish working on items list component and it's style

### CODE REVIEW

**Review by Hadeel-Salamin — APPROVED** [review]
good job :hibiscus:

File: client/src/App.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Login from './components/Login';
 import SignUp from './components/SignUp';
 import ItemDetails from './components/ItemDetails';
 import Item from './components/Item';
```
Comment (Hadeel-Salamin) [inline client/src/App.js:9]: you can remove this line (9) as it never used

File: client/src/components/ItemList/itemlist.style.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+export const List = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-content: space-between;
+`;
+
+export const StyledHeader = styled.h1`
+font-size: 20px;
```
Comment (jema28) [inline client/src/components/ItemList/itemlist.style.js:12]: There's still some problems with indentation here :/

File: client/src/components/ItemList/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    return (
+      <React.Fragment>
+        <Title />
+        <StyledHeader>Your Items</StyledHeader>
+        <List>
+          <StyledLink to="/upload-photo">+ ADD NEW ITEM</StyledLink>
+          {this.state.itemlist.map(item => <Item key={item.id} />)}
+        </List>
+        <StyledBottom>
+          <GButtonContainer><GButton title="EXPORT AS CSV" /></GButtonContainer>
```
Comment (jema28) [inline client/src/components/ItemList/index.js:29]: Please put this file in prettier, as there are indentation issues.

**Review by jema28 — CHANGES_REQUESTED** [review]
Happy with the code but formatting needs work 💄

**Review by amanshawar — APPROVED** [review]

**Pull request merged**